bitnami/discourse/templates/secrets-database.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if (include "discourse.databaseCreateSecret" .) }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "discourse.fullname" . }}-database
+  labels: {{- include "discourse.labels" . | nindent 4 }}
+  {{- if .Values.commonLabels }}
+  {{- include "discourse.tplValue" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
+  {{- end }}
+  {{- if .Values.commonAnnotations }}
+  annotations: {{- include "discourse.tplValue" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+  {{- end }}
+type: Opaque
+data:
+  {{- if and (not .Values.postgresql.enabled) (not .Values.externalDatabase.existingSecret) }}
```

Comment:
When the condition is updated, we should need to be doing this as the secret will only render in the correct scenarios

Reply:
I think you meant SHOULD NOT :) Agree. Removed.